### PR TITLE
Mustafa initial UI

### PR DESCRIPTION
**Pull request by musavic**

### CODE REVIEW

File: client/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```
Comment (shad0whe1ix) [inline client/.env:1]: what is this for?

**Review by shad0whe1ix — APPROVED** [review]
just have a quick question about the preflight skip setting. otherwise lgtm

File: client/.env (hunk 1 above)
Comment (musavic) [reply client/.env:1]: when i tried using MUI for the syling of the ui it forced me to bypass some of the warnings, difference of versions warnings between MUI and React. I havent committed all of the stylings yet, just wanted to have everything including the backend so i can get started with the algorithm

Comment (shad0whe1ix) [reply client/.env:1]: sounds good, feel free to go ahead and merge

**Pull request merged**